lib/class-wp-rest-swagger-controller.php

```diff
@@ -344,7 +351,10 @@ private function schemaIntoDefinition($schema){
 
 			
 			if($prop['type']=='array'){
-				$prop['items']=array('type'=>'string');
+				$prop['items'] = array(
+					'type'=> $prop['items']['type'],
+					'properties'=> $prop['items']['properties']
```

Comment:
No Properties in an array type? and this should also fallback to string if not set.

Reply:
Done, as above.